src/compressor/zlib/ZlibCompressor.cc

```diff
@@ -171,6 +171,7 @@ int ZlibCompressor::isal_compress(const bufferlist &in, bufferlist &out, boost::
 int ZlibCompressor::compress(const bufferlist &in, bufferlist &out, boost::optional<int32_t> &compressor_message)
 {
 #ifdef HAVE_QATZIP
+  dout(20) << "qat: " << qat_enabled << " " << __FUNCTION__ << dendl;
```

Comment:
debug message should be removed

Reply:
Thanks for your suggestion. It will be removed.